devServer.js

```diff
@@ -13,7 +13,7 @@ app.use(require('webpack-dev-middleware')(compiler, {
 
 app.use(require('webpack-hot-middleware')(compiler));
 
-app.get('/', function(req, res) {
```

Comment:
And I just realized it's still not working for me.  If I got to search page, search successfully, and then try to reload the search page, I get a "Cannot GET/search" error.  In general, problems seem to happen on reload.